packages/url/src/test/fixtures/wpt-data.json

```diff
@@ -1 +1,335 @@
\ No newline at end of file
+[
+	{ "input": "https://test:@test" },
+	{ "input": "https://:@test" },
+	{ "input": "non-special://test:@test/x" },
+	{ "input": "non-special://:@test/x" },
+	{ "input": "lolscheme:x x#x x" },
+	{ "input": "file://example:1/", "failure": true },
+	{ "input": "file://example:test/", "failure": true },
+	{ "input": "file://example%/", "failure": true },
+	{ "input": "file://[example]/", "failure": true },
+	{ "input": "http://example.com/././foo" },
+	{ "input": "http://example.com/./.foo" },
+	{ "input": "http://example.com/foo/." },
+	{ "input": "http://example.com/foo/./" },
+	{ "input": "http://example.com/foo/bar/.." },
+	{ "input": "http://example.com/foo/bar/../" },
+	{ "input": "http://example.com/foo/..bar" },
+	{ "input": "http://example.com/foo/bar/../ton" },
+	{ "input": "http://example.com/foo/bar/../ton/../../a" },
+	{ "input": "http://example.com/foo/../../.." },
+	{ "input": "http://example.com/foo/../../../ton" },
+	{ "input": "http://example.com/foo/%2e" },
+	{ "input": "http://example.com/foo/%2e%2" },
+	{ "input": "http://example.com/foo/%2e./%2e%2e/.%2e/%2e.bar" },
+	{ "input": "http://example.com////../.." },
+	{ "input": "http://example.com/foo/bar//../.." },
+	{ "input": "http://example.com/foo/bar//.." },
+	{ "input": "http://example.com/foo" },
+	{ "input": "http://example.com/%20foo" },
+	{ "input": "http://example.com/foo%" },
+	{ "input": "http://example.com/foo%2" },
+	{ "input": "http://example.com/foo%2zbar" },
+	{ "input": "http://example.com/foo%2Â©zbar" },
+	{ "input": "http://example.com/foo%41%7a" },
+	{ "input": "http://example.com/foo\t%91" },
+	{ "input": "http://example.com/foo%00%51" },
+	{ "input": "http://example.com/(%28:%3A%29)" },
+	{ "input": "http://example.com/%3A%3a%3C%3c" },
+	{ "input": "http://example.com/foo\tbar" },
+	{ "input": "http://example.com\\\\foo\\\\bar" },
+	{ "input": "http://example.com/%7Ffp3%3Eju%3Dduvgw%3Dd" },
+	{ "input": "http://example.com/@asdf%40" },
+	{ "input": "http://example.com/你好你好" },
+	{ "input": "http://example.com/‥/foo" },
+	{ "input": "http://example.com/﻿/foo" },
+	{ "input": "http://example.com/‮/foo/‭/bar" },
```

Comment:
This doesn't look right... Not sure why, searching for `oof` in my browser returns 0 results...

Same no search results in the raw diff view too https://patch-diff.githubusercontent.com/raw/WordPress/gutenberg/pull/40994.diff

Reply:
Ha, yes, unicode, today I learned:

<img width="811" alt="image" src="https://user-images.githubusercontent.com/1016458/167836024-7cfa9679-ebf5-4dd9-aac8-8ea35669ebb6.png">

Reply:
I will exclude this file from formatting since it’s autogenerated with a script anyway 👍

Reply:
Yeah, the underlined section of this code is the affected line, unicode is weird, not sure how this get's `oof` added here /shrug
![image](https://user-images.githubusercontent.com/1016458/167837973-1b0a0f5e-ac44-496f-b8be-6564abc0b803.png)

Reply:
I added this file to the list of files that won't get formatted by Prettier and removed from this PR.